src/index.js

```diff
@@ -310,7 +310,7 @@ function setupVectorLayer(glSource, accessToken, url) {
         for (let i = 0, ii = tiles.length; i < ii; ++i) {
           const tile = tiles[i];
           if (tile.indexOf('http') != 0) {
-            tiles[i] = glSource.url + tile;
+            tiles[i] = glSource.url.replace(/\/?$/, '/') + tile.replace(/^\//, '');
```

Comment:
Couldn't this be simplified?
```suggestion
            tiles[i] = glSource.url + tile.replace(/^\/?/, '/');
```

Reply:
Yes it would work but it would not have been needed if the TileJson url had been https://basemaps.arcgis.com/arcgis/rest/services/OpenStreetMap_v2/VectorTileServer/

Using `//` seems to be considered bad practice, even though it works in most cases? 

Avoiding `//` needs
`glSource.url.replace(/\/?$/, '/') + tile.replace(/^\//, '')`
or
`glSource.url.replace(/\/$/, '') + tile.replace(/^\/?/, '/')`

Reply:
Ok, point taken. What about an empty `glSource.url`? Then we'd replace a relative url with an absolute url. Maybe also not desired?

Reply:
The code is inside `if (glSource.url) {` so relative tile urls always relative to the TileJson url unless a TileJson object is specified, in which case relative paths are relative to the application,

Reply:
Alright, thanks for thinking that through with me.